app/views/content/subjects/maths/_article.html.erb

```diff
@@ -1,185 +1,138 @@
+        }
+      ],
+      colors: %w[yellow],
+      numbered: false
+    ) %>
   </section>
 
-  <section class="col col-space-l col-720">
-    <h2 class="heading-s">Some examples of what you teach 11 to 14 year olds:</h2>
+  <section class="col col-space-m col-720">
+    <h2 class="heading-s">Help to become a maths teacher</h2>
+
+    <p>Our experienced teacher training advisers can guide you through everything you need to know to apply for maths teacher training, free of charge. They can help you to:</p>
 
     <ul>
-      <li>
-        understand that the probabilities of all possible outcomes sum to 1
-      </li>
-      <li>
-        work with terminating decimals and their equivalent fractions (such as
-        0.25 and 1/4, or 3.5 and 7/2)
-      </li>
-      <li>
-        recognise, sketch and produce graphs of linear and quadratic functions.
-        Use appropriate scaling with equations in <var>x</var> and <var>y</var>
-        and the Cartesian plane
-      </li>
-      <li>
-        use Pythagoras' Theorem and trigonometric ratios in similar triangles.
-        Solve problems involving right-angled triangles
-      </li>
-      <li>
-        describe simple mathematical relationships between 2 variables in
-        observational and experimental contexts. Show this using scatter graphs
-      </li>
+      <li>understand more about maths courses</li>
+      <li>answer questions about fees and funding</li>
+      <li>make an application for a maths course</li>
     </ul>
 
-    <p>
-      <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-mathematics-programmes-of-study/national-curriculum-in-england-mathematics-programmes-of-study#key-stage-3">
-        Read the full national curriculum for more information about teaching maths in secondary schools</a>.
-    </p>
-  </section>
+    <p><a href="/teacher-training-advisers">Find out about advisers</a>.</p>
 
-  <section class="col col-space-s col-content-max">
-    <%= render TextBlockWithOneImageComponent.new(
-      heading: "Getting into teacher training",
-      colour: "yellow",
-      image_path: "static/content/hero-images/0009.jpg",
-      image_alt: "A teacher standing at the front of her classroom teaching a class",
-    ) %>
-  </section>
+    <h2 class="heading-s">Check your qualifications</h2>
+
+    <p>To train to teach maths in England, you'll need a bachelor's degree, but it does not have to be in maths.</p>
+
+    <p>Having relevant A levels can show your subject knowledge if you do not have a degree in maths.</p>
+
+    <p><a href="/is-teaching-right-for-me/qualifications-you-need-to-teach">Find out about the qualifications needed to teach</a>.</p>
 
-  <section class="col col-space-m col-720">
     <h2 class="heading-s">Funding for teaching maths</h2>
 
     <p>
     Scholarships of £29,000 or bursaries of £27,000 are available for trainee maths teachers starting their teacher training course in the 2023/24 academic year.
```

Comment:
this will need to be updated to match what's on site atm after the most recent changes